src/Symfony/Component/Routing/Tests/Loader/AnnotationClassLoaderTest.php

```diff
@@ -324,6 +324,28 @@ public function testInvokableClassWithMethodRouteLoad()
         $this->assertEquals(array_merge($classRouteData['methods'], $methodRouteData['methods']), $route->getMethods(), '->load merges class and method route methods');
     }
 
+    public function testDefaultRouteName()
+    {
+        $methodRouteData = [
+            'name' => null,
+        ];
+
+        $this->reader
+            ->expects($this->once())
+            ->method('getMethodAnnotations')
+            ->will($this->returnValue([$this->getAnnotatedRoute($methodRouteData)]))
+        ;
+
+        $routeCollection = $this->loader->load('Symfony\Component\Routing\Tests\Fixtures\AnnotatedClasses\ÀrClass');
+        $defaultName = array_keys($routeCollection->all())[0];
+
+        if (\function_exists('mb_strtolower') && preg_match('//u', $defaultName)) {
+            $this->assertSame($defaultName, 'symfony_component_routing_tests_fixtures_annotatedclasses_àrclass_routeàction');
+        } else {
+            $this->assertSame($defaultName, 'symfony_component_routing_tests_fixtures_annotatedclasses_Àrclass_routeÀction');
```

Comment:
this test is locale sensitive, this means it's not reliable
Instead, I'd suggest removing the "if" and replace it by an `@requires function mb_strtolower` annotation

Reply:
Done